<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlExplorerUI</class>
 <widget class="QMainWindow" name="SqlExplorerUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>668</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SQL Explorer</string>
  </property>
  <property name="toolTip">
   <string>SQL Explorer</string>
  </property>
  <property name="statusTip">
   <string>SQL Explorer</string>
  </property>
  <property name="whatsThis">
   <string>SQL Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="defaultTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string notr="true">Tab</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="bottomHBoxLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="databaseInfoFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="databaseLabel">
              <property name="text">
               <string>Database:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="databaseComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="tableLabel">
              <property name="text">
               <string>Table:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="tableComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="metadataToolButton">
              <property name="toolTip">
               <string>Show Metadata</string>
              </property>
              <property name="statusTip">
               <string/>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string>Metadata</string>
              </property>
              <property name="icon">
               <iconset resource="sqlexplorer.qrc">
                <normaloff>:/resources/images/metadatainfo.png</normaloff>:/resources/images/metadatainfo.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="rowsLabel">
           <property name="text">
            <string>Rows:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalRowsLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Record Count</string>
           </property>
           <property name="text">
            <string notr="true">0</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="pagingCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Paging</string>
           </property>
           <property name="statusTip">
            <string>Paging</string>
           </property>
           <property name="whatsThis">
            <string>Paging</string>
           </property>
           <property name="text">
            <string>Paging</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="pagingFrame">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="limitRowsLabel">
              <property name="text">
               <string>Limit Rows:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="limitRowsSpinBox">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>500</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="prePageToolButton">
              <property name="toolTip">
               <string>Previous Page</string>
              </property>
              <property name="text">
               <string>&lt;&lt;</string>
              </property>
              <property name="icon">
               <iconset resource="sqlexplorer.qrc">
                <normaloff>:/resources/images/previous.png</normaloff>:/resources/images/previous.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pageSpinBox">
              <property name="toolTip">
               <string>Current Page</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>/</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelTotalPages">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Total Pages</string>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="nextPageToolButton">
              <property name="toolTip">
               <string>Next Page</string>
              </property>
              <property name="text">
               <string>&gt;&gt;</string>
              </property>
              <property name="icon">
               <iconset resource="sqlexplorer.qrc">
                <normaloff>:/resources/images/next.png</normaloff>:/resources/images/next.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dbExplorerDockWidget">
   <property name="windowIcon">
    <iconset>
     <normaloff>:/resources/images/database.png</normaloff>:/resources/images/database.png</iconset>
   </property>
   <property name="toolTip">
    <string>Database Explorer</string>
   </property>
   <property name="statusTip">
    <string>Database Explorer</string>
   </property>
   <property name="whatsThis">
    <string>Database Explorer</string>
   </property>
   <property name="windowTitle">
    <string>Database Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="DatabaseExplorer" name="databaseExplorer">
    <property name="toolTip">
     <string>Database Explorer</string>
    </property>
    <property name="statusTip">
     <string>Database Explorer</string>
    </property>
    <property name="whatsThis">
     <string>Database Explorer</string>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="sqlConsoleDockWidget">
   <property name="windowIcon">
    <iconset resource="sqlexplorer.qrc">
     <normaloff>:/resources/images/console.png</normaloff>:/resources/images/console.png</iconset>
   </property>
   <property name="toolTip">
    <string>SQL Console</string>
   </property>
   <property name="statusTip">
    <string>SQL Console</string>
   </property>
   <property name="whatsThis">
    <string>SQL Console</string>
   </property>
   <property name="windowTitle">
    <string>SQL Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="SqlConsole" name="sqlConsole"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DatabaseExplorer</class>
   <extends>QWidget</extends>
   <header>databaseexplorer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SqlConsole</class>
   <extends>QWidget</extends>
   <header>sqlconsole.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="sqlexplorer.qrc"/>
 </resources>
 <connections/>
</ui>
